packages/shared/lib/core/profile/interfaces/chrysalis-persisted-profile.interface.ts

```diff
@@ -18,7 +18,7 @@ export interface IChrysalisPersistedProfile {
     hasVisitedStaking?: boolean
     lastShimmerPeriodVisitedStaking?: number
     lastAssemblyPeriodVisitedStaking?: number
-    lastUsedAccountId?: string
+    lastUsedAccountId?: string // TODO(2.0) Accounts are gone
```

Comment:
Why does this comment come back? 😢 
I think there is some error with some merge conflict haha
This comment should not be here